decentraland/unity-explorer
# Fix: camera-reel screencapture sidebar button being on top layout

## What does this PR change?

Separates layout into top and bottom layouts and moved screencapture button in the bottom one.

## How to test the changes?

1. Launch the explorer with proper CLI argument
2. Verify that camera reel screencapture button is down on the sidebar (near to emotes button)
3. Verify this button is not presented if you run it without CLI args

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
